apps/src/applab/designMode.js

```diff
@@ -135,6 +135,11 @@ designMode.onPropertyChange = function(element, name, value) {
     case 'width':
       element.style.width = value + 'px';
       element.parentNode.style.width = value + 'px';
+
+      if (element.style.backgroundSize) {
```

Comment:
I am concerned that if I:
1. create an element 80px wide x 40px high
2. increase the width to 100px
3. increase the height to 50px

then I will have a 100px by 50px image with a 100px by 40px background. is this a problem? would it make sense to do this update when height is changed too?